### DISCUSSION TITLE
Add CATEGORY_APP_WEATHER to the manifest

### DISCUSSION THREAD

**Discussion by devycarol** [breezy-weather/breezy-weather#1238] [Ideas]
### Describe your idea

I think "android.intent.category.APP_WEATHER" should be added to the manifest. This intent category was added in the Tiramisu API, and it would be useful to include this as this app would be listed among as a "weather app" when relevant. It also might be relevant metadata for app stores—not an expert there.

### Other details

If your curious why I care about this, I ran into this when developing a weather command for my [digital assistant app](https://github.com/devycarol/Emilla), which you're welcome to check out if you like :)

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.6 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Would you be willing to make the relevant pull request?
I'm not exactly sure how to implement and test your use case ;)

**Reply by devycarol** [reply]
It's been done :)
